components/Premium/Premium.tsx

```diff
@@ -138,6 +138,7 @@ function Premium() {
                             </Modal.Body>
                         </Modal>
                     </h2>
+                    <p>By buying one of the following products, you confirm the immediate execution of the contract, hereby losing your cancellation right.</p>
```

Comment:
```suggestion
                    <p>By purchasing one of the following products, you confirm the immediate execution of the contract and thus lose your right of withdrawal.</p>
```
I am pretty sure that the user has to explicitly confirm that he want it to execute immediately (via a checkbox) although I think the confirmation can be saved and doesn't have to be repeated

Reply:
oh, I am sorry apparently this wasn't saved with my last review. Explains why you were so confused :neutral_face: